fuseml/questions.yaml

```diff
@@ -8,3 +8,24 @@ questions:
   type: string
   label: Path to custom repository with FuseML extensions
   group: "Global Options"
+- variable: extensions.mlflow
+  default: false
+  required: false
+  type: boolean
+  label: Install MLFlow extension
+  description: "MLFlow is an open source platform specialized in tracking ML experiments, and packaging and deploying ML models."
+  group: "Extensions"
+- variable: extensions.kfserving
+  default: false
+  required: false
+  type: boolean
+  label: Install KFServing extension
+  description: "Serverless Inferencing on Kubernetes"
+  group: "Extensions"
+- variable: extensions.custom
```

Comment:
Would you please consider adding check boxes for the other available options (ovms, seldon core) ? The more, the merrier.

Reply:
I did not, because currently the chart is targeting latest stable version of fuseml, which does not have them...